docs/sources/developers/plugins/migration-guide.md

```diff
@@ -308,6 +319,32 @@ import { cx, css } from 'emotion';
 import { cx, css } from '@emotion/css';
 ```
 
+### Update needed for app plugins using dashboards
+
+To make side navigation work properly - app plugins targeting Grafana 8.+ and using navigation bar need to adjust their `plugin.json` and all dashboard json files to have a matching `uid`.
+
+```json
+
+plugins.json
+{
+  "type": "dashboard",
+  "name": "(Team) Situation Overview",
+  "path": "dashboards/example-dashboard.json",
+  "addToNav": true,
+  "defaultNav": false,
+  "uid": "l3KqBxCMz"
+}
+
+dashboards/example-dashboard.json
+
+{
+  // excerpt of dashboard content
+  "title": "Example Dashboard",
+  "uid": "l3KqBxCMz",
+  "version": 1
+}
+```
```

Comment:
**Nit:** maybe formatting like this instead (to highlight the changes)?

**`plugin.json`**
```diff
{
  "type": "dashboard",
  "name": "(Team) Situation Overview",
  "path": "dashboards/example-dashboard.json",
  "addToNav": true,
  "defaultNav": false,
+  "uid": "l3KqBxCMz"
}
```

**`dashboards/example-dashboard.json`**
```diff
{
  "title": "Example Dashboard",
+  "uid": "l3KqBxCMz",
  "version": 1
  // ...
}
```
```

Reply:
Sorry, probably this won't really work in [Hugo](https://gohugo.io/) as I imagined, just ignore my comment.